Mathlib/Analysis/RCLike/Basic.lean

```diff
@@ -1150,3 +1150,32 @@ noncomputable def IsRCLikeNormedField.rclike (𝕜 : Type*)
   exact p.copy_of_normedField hk hp
 
 end
+
+open Bornology Metric Set
+open scoped Topology
+
+variable {G : Type*} [SeminormedAddCommGroup G] [TopologicalAddGroup G] {s : Set G}
+
+/-- A group hom from a normed group to `ℝ` or `ℂ`, bounded on a neighborhood of `0`, must be
+continuous. -/
+lemma AddMonoidHom.continuous_of_isBounded_nhds_zero (f : G →+ K) (hs : s ∈ 𝓝 (0 : G))
```

Comment:
Taking as your target just R or C looks very restrictive, no? Isn't it enough to have a normed vector space over a nontrivially normed field?

Reply:
I don't really see how to generalise to nontrivially normed fields because I need a *natural number* `n` such that `‖(n : K)⁻¹‖` is arbitrarily small. The fact that it must be a natural number comes from the fact that I'm talking about group homs, not eg `K`-linear maps.

I could however generalise to `ℝ`- or `ℂ`-normed spaces